client/verta/verta/_internal_utils/model_dependencies.py

```diff
@@ -71,3 +71,14 @@ def class_module_names(model_class: Type[VertaModelBase]) -> Set[str]:
         mods_in_signature = modules_in_function_signature(_func)
         modules_found |= mods_in_body | mods_in_signature
     return modules_found
+
+
+# TODO VRD-682 convert module names to pkg names here
+def package_names(module_names: Set[str]) -> Set[str]:
+    """Return a set of all packages detected in the provided set of base module names."""
+    return module_names
+
+
+def strip_version(pkg_str: str) -> str:
+    """Return the base package name from a package string."""
+    return pkg_str.split('==')[0]
```

Comment:
I've looked through the functions in `_pip_requirements_utils` and none of them seems to do exactly what we need here.  Given the string `pandas==1.0.6`, we need back the package name.  `parse_version` parses the version bit (`1.0.6`) and `remove_pinned_requirements` isn't it either.  Would it be better to add a new utility here, rather than with the `model_dependenies` utilities?

Reply:
Oops a bad copy and paste on my part. I meant `parse_req_spec()`